MODULE.bazel

```diff
@@ -22,20 +22,20 @@ IO_GRPC_GRPC_JAVA_ARTIFACTS = [
     "com.google.truth:truth:1.1.5",
     "com.squareup.okhttp:okhttp:2.7.5",
     "com.squareup.okio:okio:2.10.0",  # 3.0+ needs swapping to -jvm; need work to avoid flag-day
-    "io.netty:netty-buffer:4.1.100.Final",
-    "io.netty:netty-codec-http2:4.1.100.Final",
-    "io.netty:netty-codec-http:4.1.100.Final",
-    "io.netty:netty-codec-socks:4.1.100.Final",
-    "io.netty:netty-codec:4.1.100.Final",
-    "io.netty:netty-common:4.1.100.Final",
-    "io.netty:netty-handler-proxy:4.1.100.Final",
-    "io.netty:netty-handler:4.1.100.Final",
-    "io.netty:netty-resolver:4.1.100.Final",
+    "io.netty:netty-buffer:4.1.109.Final",
+    "io.netty:netty-codec-http2:4.1.109.Final",
+    "io.netty:netty-codec-http:4.1.109.Final",
+    "io.netty:netty-codec-socks:4.1.109.Final",
+    "io.netty:netty-codec:4.1.109.Final",
+    "io.netty:netty-common:4.1.109.Final",
+    "io.netty:netty-handler-proxy:4.1.109.Final",
+    "io.netty:netty-handler:4.1.109.Final",
+    "io.netty:netty-resolver:4.1.109.Final",
     "io.netty:netty-tcnative-boringssl-static:2.0.61.Final",
     "io.netty:netty-tcnative-classes:2.0.61.Final",
```

Comment:
tcnative should also be bumped to version 2.0.65 to match the bom for Netty 4.1.109

Reply:
Netty upgrade requires running more tests. We are doing it in https://github.com/grpc/grpc-java/pull/11273.